theme/styles/settings/_vars.scss

```diff
@@ -1,7 +1,7 @@
 /** Fonts */
 
 $itc-garamond: ITCGaramondLight, serif;
```

Comment:
Not important, but one of the reasons why it can be nice to label font variables `$serif` `$sans-serif` instead of by the name of the actual font: you can just change it in one place, and it will propagate everywhere :)

Reply:
Good idea, in case there is another font change! I'll do this in another PR so this one isn't cluttered with those file changes ;)